engine/platform/src/platform_window_glfw.cpp

```diff
@@ -154,6 +154,45 @@ namespace dmPlatform
         return 0;
     }
 
+    static void OpenGLVersionHintToMajorMinor(PlatformOpenGLVersion version, uint32_t* major, uint32_t* minor)
```

Comment:
It's static but I don't see this being used in this file?
Also, it feels like the code should be in one place perhaps?